frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx

```diff
@@ -99,34 +182,10 @@ class AddSubCategory extends React.Component<
             <Row>
```

Comment:
The title should be changed according to the component's type

Reply:
ok